<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnowModeling</class>
 <widget class="QMainWindow" name="SnowModeling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Моделирование снега</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Canvas" name="m_canv" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>861</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxIntensity">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>100</y>
      <width>211</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Интенсивность осадков</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QSlider" name="hsldIntensity">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>22</width>
       <height>241</height>
      </rect>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblMax">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblHard">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblMiddle">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblMin">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblNothing">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
